pkg/obs/services/buckets.go

```diff
@@ -235,8 +234,8 @@ func ListBuckets(c *config.CloudCredentials, opts ListOpts) ([]*Bucket, error) {
 	for k, j := 0, i+1; j <= i+opts.Limit && j < len(output.Buckets); k, j = k+1, j+1 {
 		func(bucketName string, idx int) {
 			group.Go(func() error {
-				fsBucket, err := GetParallelFSBucket(c, bucketName)
-				bucketList[idx] = fsBucket
+				obsBucket, err := GetObsBucket(c, bucketName)
+				bucketList[idx] = obsBucket
 				return err
```

Comment:
```suggestion
				obsBucket, err := GetObsBucket(c, bucketName)
				if err != nil {
					return err
				}
				
				bucketList[idx] = obsBucket
				return nil
```

Reply:
The logic is the same, if err not empty, `bucketList` still not be return.
Here only change function name to `GetObsBucket`. It is better to keep the original logic